From eab9b813e6fd14dcf28eddf41afd303eefef6fc1 Mon Sep 17 00:00:00 2001
Message-Id: <eab9b813e6fd14dcf28eddf41afd303eefef6fc1.1423872525.git.feitong.yi@intel.com>
In-Reply-To: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
References: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
From: "Kumar, Kiran S" <kiran.s.kumar@intel.com>
Date: Wed, 4 Feb 2015 17:36:46 +0530
Subject: [PATCH 07/95] SQUASHME! [VPG]: drm/i915: Rectified pipe number in
 CSC programming

SQUASHME! - This patch should be squashed into the following existing patch:

        Author: Kumar, Mahesh <mahesh1.kumar@intel.com>
        Date:   Fri Nov 14 09:52:04 2014 +0530
        Change-Id: I082d6ce8cf655b90f9a8c451ad237953f785a146
        REVERTME [VPG]: drm/i915: Add support for Pipe C gamma, csc and hue

Pipe number was not correct for pipe 2, Rectified the same.
Removed an incorrect return while restoring the color state.

Issue: GMINL-5638
Change-Id: Icb33c1a7ac88ac4984a54ab0b1e77362009d4d10
Signed-off-by: Kumar, Kiran S <kiran.s.kumar@intel.com>
---
 drivers/gpu/drm/i915/i915_sysfs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_sysfs.c b/drivers/gpu/drm/i915/i915_sysfs.c
index 815ca66..9804a0b 100644
--- a/drivers/gpu/drm/i915/i915_sysfs.c
+++ b/drivers/gpu/drm/i915/i915_sysfs.c
@@ -694,7 +694,7 @@ static ssize_t csc_enable_show(struct device *kdev,
 		dev_priv->csc_enabled[1] ? "Enabled" : "Disabled");
 	if (IS_CHERRYVIEW(dev))
 		len += scnprintf(ubuf+len, PAGE_SIZE, "Pipe 2: %s\n",
-			dev_priv->csc_enabled[1] ? "Enabled" : "Disabled");
+			dev_priv->csc_enabled[2] ? "Enabled" : "Disabled");
 
 	return len;
 }
-- 
1.7.9.5

